app/products/calls/screens/host_controls/host_controls.tsx

```diff
@@ -42,8 +55,12 @@ export const HostControls = ({
 }: Props) => {
     const intl = useIntl();
     const {bottom} = useSafeAreaInsets();
+    const theme = useTheme();
+    const {openUserProfile} = useHostMenus();
+    const styles = getStyleFromTheme(theme);
 
     const sharingScreen = currentCall.screenOn === session.sessionId;
+    const displayName = displayUsername(session.userModel, intl.locale, teammateNameDisplay, true);
```

Comment:
perhaps can be memoized to not compute all the time? up to you

Reply:
Cool, I put it in the callback cause it's only being used there.